src/Components/App/index.js

```diff
@@ -16,8 +15,9 @@ export default class App extends Component {
         <Navbar />
         <AuthRequired>
           <Switch>
-            <Route path="/login" component={Login} />
-            <Route component={Welcome} />
+            <Route path="/login" component={ Login } />
+            <Route component={ Register } />
```

Comment:
mogelijk hier even een path toevoegen, zodat we de productregistratie bijvoorbeeld alleen laten zien op `/product/register`

Reply:
Done